src/bilder/images/keycloak/files/docker-compose.yaml

```diff
@@ -23,6 +23,8 @@ services:
     image: traefik:v2.10
     command:
     - "--configFile=/etc/traefik/traefik.yaml"
+    labels:
+    - "traefik.http.middlewares.keycloak-ratelimit.ratelimit.average=10"
```

Comment:
Do we want this applied at the Traefik layer, or in the Keycloak service settings?

Reply:
Don't think it matters in this case given that it's only servicing keycloak in this case.